cmd/xl-v1-metadata.go

```diff
@@ -173,7 +176,8 @@ func newXLMetaV1(object string, dataBlocks, parityBlocks int) (xlMeta xlMetaV1)
 // IsValid - tells if the format is sane by validating the version
 // string and format style.
 func (m xlMetaV1) IsValid() bool {
```

Comment:
There is no problem with that.  We are going to read both we shouldn't fail. We don't fail in xl either.  We should read all files.. in newer version. 

We only fail for format.json because directory structure was not recognized.